packages/components/pie-button/playwright-lit-visual.config.ts

```diff
@@ -33,11 +33,20 @@ export default defineConfig({
   /* Configure projects for major browsers */
   projects: [
     {
-      name: 'component:chrome',
+      name: 'visual:desktop',
+      grepInvert: /@mobile/,
       use: {
         ...devices['Desktop Chrome'],
       },
-      testMatch: ['**/test/visual/*.spec.ts']
-    }
+      testMatch: ['**/test/visual/*.spec.js']
```

Comment:
Sorry, just realised we're using TypeScript in the pie-button test specs, so this glob should be `**/test/visual/*.spec.ts`